server/runtime/src/main/java/org/infinispan/server/state/ServerStateManagerImpl.java

```diff
@@ -0,0 +1,284 @@
+      cache.addListener(new IgnoredCachesListener(), new ScopeFilter(IGNORED_CACHES_KEY.getScope()), null);
+      cache.addListener(new ConnectorStateListener(), new ScopeFilter(CONNECTOR_STATE_SCOPE), null);
+      cache.addListener(new ConnectorIpFilterListener(), new ScopeFilter(CONNECTOR_IPFILTER_SCOPE), null);
+   }
+
+   @Override
+   public CompletableFuture<Void> unignoreCache(String cacheName) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      synchronized (this) {
+         ignored.caches.remove(cacheName);
+         hasIgnores = !ignored.caches.isEmpty();
+         return cache.putAsync(IGNORED_CACHES_KEY, ignored).thenApply(r -> null);
+      }
+   }
+
+   @Override
+   public CompletableFuture<Void> ignoreCache(String cacheName) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      synchronized (this) {
+         ignored.caches.add(cacheName);
+         hasIgnores = !ignored.caches.isEmpty();
+         return cache.putAsync(IGNORED_CACHES_KEY, ignored).thenApply(r -> null);
+      }
+   }
+
+   @Override
+   public Set<String> getIgnoredCaches() {
+      return Collections.unmodifiableSet(ignored.caches);
+   }
+
+   @Override
+   public boolean isCacheIgnored(String cacheName) {
+      return hasIgnores && ignored.caches.contains(cacheName);
+   }
+
+   @Override
+   public CompletableFuture<Boolean> connectorStart(String name) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      return cache.removeAsync(new ScopedState(CONNECTOR_STATE_SCOPE, name), true).thenApply(v -> null);
+   }
+
+   @Override
+   public CompletableFuture<Void> connectorStop(String name) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      return cache.putAsync(new ScopedState(CONNECTOR_STATE_SCOPE, name), true).thenApply(v -> null);
+   }
+
+   @Override
+   public CompletableFuture<Boolean> connectorStatus(String name) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      return CompletableFuture.completedFuture(server.getProtocolServers().get(name).getTransport().isRunning());
+   }
+
+   @Override
+   public CompletableFuture<Void> setConnectorIpFilterRule(String name, Collection<IpSubnetFilterRule> filterRule) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      IpFilterRules ipFilterRules = new IpFilterRules();
+      filterRule.forEach(r -> ipFilterRules.rules.add(new IpFilterRule(r)));
+      return cache.putAsync(new ScopedState(CONNECTOR_IPFILTER_SCOPE, name), ipFilterRules).thenApply(v -> null);
+   }
+
+   @Override
+   public CompletableFuture<Void> clearConnectorIpFilterRules(String name) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      return cache.removeAsync(new ScopedState(CONNECTOR_IPFILTER_SCOPE, name)).thenApply(v -> null);
```

Comment:
[usability]
you should put an empty list instead of removing the entry

Use case:
* configure IP-filter in infinispan.xml
* start server1
* clear IP-filter in server1
* start server2 (same config as infinispan.xml)

server2 contains the IP-filter configured in infinispan.xml instead of disabled IP-filter.